portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,37 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="cover">
+      <h>Annie Yang</h>
+    </div>
+    <div class="topnav" id="myTopNav">
+      <a href="#home" class="active">Home</a>
+      <a href="#about">About me</a>
+      <a href="#project">Projects</a>
+      <a href="#hobbies">What I enjoy</a>
+      <a href="#gallery">Gallery</a> 
+      <a href="#blog">Blog Posts</a>
+    </div>
+    <div class="about">
+      <h1>About Me</h1>
+      <p>Hi! My name is Annie and I’m currently a second year Software Engineering student at UNSW and a STEP Intern at Google 2020. 
+        I’m into software development, especially in the backend side and security engineering. 
+        My passion for technology and software stemmed from my curiosity and obsession with video games when I was younger. 
+        I was always curious into how games were developed and really wanted to make one myself. That was what put me into a path of Software Engineering. 
+        Nowadays I still spend a lot of my free time playing video games, however I have shifted my career interest from games to general software programming.</p>
+    </div>
```

Comment:
Did you run this by clang-format ?

Reply:
Yes, I just ran clang-format and it is all good!